src/hubmap_translation/addl_index_transformations/portal/add_assay_details.py

```diff
@@ -37,6 +38,10 @@ def add_assay_details(doc, transformation_resources):
         doc['mapped_data_types'] = [assay_details.get('description')]
```

Comment:
If the soft-assay endpoint returns an empty json for this [example (58ebb89caf1512e9452d1f9e0e1efa8e)](https://ingest-api.dev.hubmapconsortium.org/assaytype/58ebb89caf1512e9452d1f9e0e1efa8e), do we end up with `[None]`?

```suggestion
        doc['mapped_data_types'] = [assay_details.get('description', doc.get('dataset_type'))]
```

Reply:
Empty json triggers line 24 which ultimately returns `{'description': dataset_type, 'vitessce-hints': ['unknown-assay'], 'error': empty_error_msg}`.